cloud/profitbricks/profitbricks_volume.py

```diff
@@ -308,19 +328,52 @@ def delete_volume(module, profitbricks):
 
     return changed
 
+
+def _attach_volume(module, profitbricks, datacenter, volume):
+    """
+    Attaches a volume.
+
+    This will attach a volume to the server.
+
+    module : AnsibleModule object
+    profitbricks: authenticated profitbricks object.
+
+    Returns:
+        True if the volume was attached, false otherwise
+    """
+    server = module.params.get('server')
+
+    # Locate UUID for Server
+    if server:
+        if not (uuid_match.match(server)):
+            server_list = profitbricks.list_servers(datacenter)
+            for s in server_list['items']:
+                if server == s['properties']['name']:
+                    server = s['id']
+                    break
+
+        try:
+            return profitbricks.attach_volume(datacenter, server, volume)
+        except Exception as e:
+            module.fail_json(msg='failed to attach volume: %s' % str(e))
+
+
 def main():
     module = AnsibleModule(
         argument_spec=dict(
             datacenter=dict(),
+            server=dict(),
             name=dict(),
-            size=dict(default=10),
+            size=dict(type='int', default=10),
             bus=dict(default='VIRTIO'),
```

Comment:
could you please also add a choice list for bus?
~~~python
bus=dict(default='VIRTIO', choices=['IDE', 'VIRTIO']),
~~~